README.md

```diff
@@ -906,19 +906,21 @@ You can create, update, delete or load roles:
 
 ```typescript
 // You can optionally set a description and associated permission for a roles.
+// setting tenant id - will create this role for this specific tenant scope, leave empty to create a role taht will be allowed on all tenants
```

Comment:
small typo, also some small recommendation 

```suggestion
// Set the tenant ID to create a role scoped to a specific tenant. Leave empty for a global role applicable across all tenants.
```

Reply:
Took the other suggestion